ciphers/simple_keyword_cypher.py

```diff
@@ -21,7 +21,7 @@ def create_cipher_map(key: str) -> dict[str, str]:
     :param key: keyword to use
     :return: dictionary cipher map
     """
-    # Create alphabet list
+    # Create an alphabet list
```

Comment:
```suggestion
    # Create a list of the letters in the alphabet
```
In my opinion, this looks better but I'm open to external opinions